.github/workflows/gradle-check.yml

```diff
@@ -92,12 +99,7 @@ jobs:
           issue-number: ${{ env.pr_number }}
           body: |
             ### Gradle Check (Jenkins) Run Completed with:
-            * **RESULT:** ${{ env.result }} :grey_exclamation:
-            * **FLAKY TEST FAILURES:**
-            The following tests failed but succeeded upon retry:
-            ```
-            ${{ env.test_failures }}
-            ```
+            * **RESULT:** ${{ env.result }} :grey_exclamation: ${{ env.test_failures }}
```

Comment:
Can we do this same refactoring for the "Create Comment Failure" step as well?

Reply:
Yes, my bad. Will do.